examples/drivers/windows/otLwf/radio.c

```diff
@@ -759,6 +759,13 @@ ThreadError otPlatRadioEnergyScan(_In_ otInstance *otCtx, uint8_t aScanChannel,
     return NT_SUCCESS(status) ? kThreadError_None : kThreadError_Failed;
 }
 
+void otPlatRadioSetDefaultTxPower(otInstance *aInstance, int8_t aPower)
+{
+    // TODO: Create a proper implementation for this driver.
+    (void)aInstance;
```

Comment:
Thanks for pointing that I out. I didn't realize it had a property already. Could you add the following code to this function then?

```
    NT_ASSERT(otCtx);
    PMS_FILTER pFilter = otCtxToFilter(otCtx);
    NTSTATUS status;

    // Indicate to the miniport
    status =
        otLwfCmdSetProp(
            pFilter,
            SPINEL_PROP_PHY_TX_POWER,
            SPINEL_DATATYPE_INT8_S,
            aPower
        );
    if (!NT_SUCCESS(status))
    {
        LogError(DRIVER_DEFAULT, "Set SPINEL_PROP_PHY_TX_POWER failed, %!STATUS!", status);
    }
```